### ISSUE TITLE
overflow property has been removed from Stack in latest master

### ISSUE THREAD

**Issue by esDotDev** [ReinBentdal/styled_widget#51]
I'm getting these errors in the latest master: 
```
extensions/list_extension.dart(63,9): error GBF4691A2: No named parameter with the name 'overflow'. 
````

**Comment by ReinBentdal** [comment]
That Is strange. Would appreciate if you could open a pull request to include this method again

**Comment by esDotDev** [comment]
Sorry, it would be more clear to say overflow has been replaced by Stack.clipBehavior
https://api.flutter.dev/flutter/widgets/Stack/clipBehavior.html

They have been planning to remove it for a while apparently:
https://api.flutter.dev/flutter/widgets/Stack/overflow.html
_This overrides clipBehavior for now due to a staged roll out without breaking Google. We will remove it and only use clipBehavior soon_

**Comment by alejandroaap** [comment]
I have the same problem. To fix it I had to use an older version of flutter. When are you going to release a library update?

**Comment by ReinBentdal** [comment]
The issue is resolved in #52. Will try to make an update today

**Comment by ReinBentdal** [comment]
The update is now available as version 0.2.2
